zer0-os/zOS
# feat(conversations): change header

### What does this do?

- Gets rid of the `Conversations` header
- Moves the `New Conversation` button to the right of the search bar 

![Screenshot 2023-06-28 at 4 02 47 PM](https://github.com/zer0-os/zOS/assets/12437916/b927d5ec-e074-4280-86b0-3c2cfe14a509)

### Why are we making this change?

Design changed slightly

### How do I test this?

1. Open the app
2. Open messenger
3. Check that the conversation search UI matches Figma

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
